solidity/contracts/interfaces/IInterchainSecurityModule.sol

```diff
@@ -9,7 +9,8 @@ interface IInterchainSecurityModule {
         LEGACY_MULTISIG,
         MERKLE_ROOT_MULTISIG,
         MESSAGE_ID_MULTISIG,
-        OPTIMISM
+        OPTIMISM,
```

Comment:
FYI https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2420/files#diff-7d0323c3aa18eb73dd096474183d65c5d2495f0ea62685db5e6f7117c96e63fcR12

Reply:
Oh yep, which do we think will be merged first? This or that PR

Reply:
There is no rush on either of them, asked @yorhodes to take a peek at this one before we merge